getting_started/scripting/gdscript/gdscript_styleguide.rst

```diff
@@ -194,3 +207,36 @@ Constants
 
 Use CONSTANT\_CASE, all caps, with an underscore (\_) to separate words:
 ``const MAX_SPEED = 200``
+
+Static Typing
+-------------
+
+Since Godot 3.1, GDScript supports :ref:`optional static typing<_doc_gdscript_static_typing>`.
+
+Type hints
+~~~~~~~~~~
+
+Place the colon right after the variable's name, without a space, and let the GDScript compiler infer the variable's type when possible.
+
+**Good**:
+
+::
+
+   onready var health_bar: ProgressBar = get_node("UI/LifeBar")
+
+   var health: = 0 # The compiler will use the int type
```

Comment:
Most code I've seen writes this as `name := value`, I suppose to group symbols together and in a similar vain to `name += value`, but `name: = value` would be more consistent with `name: type = value`. I suppose it would be nice to get community feedback on this.

Reply:
I think the recommended style should be `name := value` indeed.